src/excalidraw-app/collab/RoomDialog.tsx

```diff
@@ -31,8 +32,20 @@ const RoomDialog = ({
   setErrorMessage: (message: string) => void;
 }) => {
   const roomLinkInput = useRef<HTMLInputElement>(null);
-  const navigator = window.navigator as any;
-  const isAppleBrowser = /Apple/.test(navigator.vendor);
+
+  const getShareIcon = () => {
```

Comment:
since it is a pure function lets keep it outside`RoomDialog`  so its not recreated (new reference) for every rerender.

Reply:
Oh, right, good catch, @ad1992!